examples/resource_lacework_alert_channel_datadog/main.tf

```
@@ -14,8 +14,9 @@ variable "channel_name" {
 }
 
 resource "lacework_alert_channel_datadog" "example" {
-  name            = var.channel_name
-  datadog_site    = "eu"
-  datadog_service = "Events Summary"
-  api_key         = "datadog-key"
+  name             = var.channel_name
+  datadog_site     = "eu"
+  datadog_service  = "Events Summary"
+  api_key          = "datadog-key"
+  test_integration = false
```

Could we please add a message something like this one? I don't
want users to do this by mistake.
```suggestion

  // test_integration input is used in this example only for testing
  // purposes, it help us avoid sending a "test" request to the 
  // system we are integrating to. In production, this should remain
  // turned on ("true") which is the default setting
  test_integration = false
```